src/packages/dom.js

```diff
@@ -0,0 +1,131 @@
+let InitialTemp;
+
+const container = document.querySelector('#container');
+
+const DOMGetElement = (selector) => document.querySelector(selector);
+
+const DOMDismisMessage = () => {
+  setTimeout(() => {
+    DOMGetElement('.msg').remove();
+  }, 3000);
+};
+
+const displayForm = () => {
+  const form = `
+  <form id="form">
+    <h2>Find the weather of your city!!</h2>
+    <div class="form-group">
+      <input type="text" placeholder="Enter Location" id="location">
+      <input type="submit" value="Search">
+    </div>
+  </form>
+  `;
+  container.innerHTML = form;
+};
+
+const setBackGround = (background) => {
+  container.style.background = `url("${window.URL.createObjectURL(
+    background,
+  )}") top/cover`;
+  container.classList.add('after');
+  DOMGetElement('.fa-spinner').remove();
+  if (!DOMGetElement('#form')) {
+    displayForm();
+  }
+};
+
+const DOMDisplayMessage = (msg, type) => {
+  const error = `
+    <div class="${type} msg">
+      <span>${msg}</span>
+    </div>
+  `;
+  container.insertAdjacentHTML('beforeend', error);
+  DOMDismisMessage();
+};
+
+const setSpinner = () => {
+  container.insertAdjacentHTML('beforeend', '<i class="fas fa-spinner"></i>');
+};
+
+const DOMGetInputValue = (selector) => document.querySelector(selector).value;
+
+const setDefatulBackground = () => {
+  container.classList.add('default');
+  container.classList.add('after');
+  DOMGetElement('.fa-spinner').remove();
+  if (!DOMGetElement('#form')) {
+    displayForm();
+  }
+};
+
+const DOMChangeTemp = (temp, target) => {
+  DOMGetElement('.tempNum span').innerHTML = temp;
+  DOMGetElement('.active').classList.remove('active');
+  target.classList.add('active');
+};
+
+const toogleTemp = (e, initial) => {
+  if (e.target.getAttribute('id') === 'celsius') {
+    DOMChangeTemp(initial, e.target);
+  } else {
+    DOMChangeTemp(((initial * (9 / 5)) + 32), e.target);
+  }
+};
```

Comment:
The application makes different requests for the Celsius data and the Fahrenheit data. While using math is great and efficient! Using it for this assignment beats the point of the assignment.

This was mentioned in Pathwright instructions. Please see this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/requests_requirement.png) of same.